README.md

```diff
@@ -103,6 +103,29 @@ However, this can be enabled using the following steps:
 - Click on the Preview new design button.
 - Enable the `Synchronize site fixes` setting under `Settings -> Manage Settings`.
 
+## Pywal colorscheme integration for Firefox
+
+1. In FireFox's <code>about:config</code> set <code>toolkit.legacyUserProfileCustomizations.stylesheets</code> to <code>true</code>.
+2. Find your profile's root directory in <code>about:profiles</code> and (if it doesn't already exist) create a new folder inside it called <strong>chrome</strong>.
+3. Locate [Pywal's](https://github.com/dylanaraps/pywal) <strong>colors.css</strong>. By default this should be located at <code>~/.cache/wal/colors.css</code>
+4. Hard link this file to inside the <code>chrome</code> directory you just created using either:
+   - <strong>userContent.css</strong> as your link name if this file does not already exist the folder.
+   - <strong>colors.css</strong> as your link name if <strong>userContent.css</strong> already exists. Additionally, add <code>@import "colors.css";</code> to the top of your <strong>userContent.css</strong>.
```

Comment:
I think this uses vague wording. What's hardlinking?

Reply:
Would putting in the command required be better?
`ln /location/of/colors.css /location/of/chrome/colors.css`
As well as a similar windows command?

Reply:
I think it's better to say "create a symlink" and for those that don't know can simply search "how to create symlink <platform>" and find a dozen tutorials

Reply:
A lot of the tutorials I saw treated symlinks and hard links different things (i.e. `ln -s` vs `ln` respectively). Using `ln -s` creates a softlink which I can't get firefox to read.

Reply:
Maybe don't let dark reader be the one telling this? We simply just read whatever Firefox gives us. If a user wants to link their existing css generated by whatever tool, 🤷🏽.

Reply:
I guess I could create my own lil guide or add something to the pywal wiki. I did create this with its main intention to have it work with pywal.

Reply:
Yeah that sounds like the best solution, this is a bit out-of-league for Dark Reader IMO.